src/Paket/Paket.fsproj

```diff
@@ -31,6 +31,14 @@
     <StartAction>Project</StartAction>
     <StartProgram>paket.exe</StartProgram>
     <StartAction>Project</StartAction>
+    <StartArguments>restore --project "C:\source\Paket\integrationtests\scenarios\i00xxxx-obj-dir\temp\ObjDir\ObjDir.csproj" --target-framework net461 --new-sdk-references-file "C:\source\Paket\integrationtests\scenarios\i00xxxx-obj-dir\temp\ObjDir\MyCustomFancy\xxx.paket.ref" --intermediate-dir "C:\source\Paket\integrationtests\scenarios\i00xxxx-obj-dir\temp\ObjDir\MyCustomFancy"</StartArguments>
+    <StartWorkingDirectory>C:\source\Paket\integrationtests\scenarios\i00xxxx-obj-dir\temp\ObjDir</StartWorkingDirectory>
+    <StartArguments>restore</StartArguments>
+    <StartWorkingDirectory>C:\temp\perf\</StartWorkingDirectory>
+    <StartArguments>update</StartArguments>
+    <StartWorkingDirectory>C:\proj\testing\testpaketfailure\</StartWorkingDirectory>
+    <StartArguments>fix-nuspecs files src/Paket.Core.preview3/obj/Paket.Core.1.0.0.nuspec project-file src/Paket.Core.preview3/Paket.Core.fsproj</StartArguments>
+    <StartWorkingDirectory>C:\proj\Paket</StartWorkingDirectory>
```

Comment:
I do this myself all the time. I don't know why vs is failing so hard here.

Ideally we would gitignore that section of this file

Reply:
yeah ... please remind me again after I remove the wip